File: FBOLinx.Web/FBOLinx.Core/Enums/DocumentsEnum.cs
using System.ComponentModel;

namespace FBOLinx.Core.Enums
{
    public enum DocumentAcceptanceFlag : short
    {
        [Description("Force Accepted")]
        ForceAccepted = 0,
        [Description("Accepted")]
        Accepted = 1,
        [Description("Not Accepted Required")]
        NotAcceptedRequired = 2,
    }
    public enum DocumentTypeEnum : short
    {
        [Description("EULA")]
        EULA = 0,
        [Description("Cookie Policy")]
        Cookie = 1,
        [Description("Privacy Policy")]
        Privacy = 2,
    }
}

File: FBOLinx.Web/FBOLinx.Web/ClientApp/src/app/enums/documents..enum.ts
export enum DocumentAcceptanceFlag {
    ForceAccepted = 0,
    Accepted = 1,
    NotAcceptedRequired = 2
}

export enum DocumentTypeEnum {
    EULA = 0,
    Cookie = 1,
    Privacy = 2
}

export const DocumentTypeEnumDescription: Record<DocumentTypeEnum, string> = {
    [DocumentTypeEnum.EULA]: "EULA",
    [DocumentTypeEnum.Cookie]: "Cookie Policy",
    [DocumentTypeEnum.Privacy]: "Privacy Policy"
  };
